modules/@apostrophecms/util/ui/src/http.js

```diff
@@ -99,6 +99,9 @@ export default () => {
         throw new Error('If you wish to receive a promise from apos.http methods in older browsers you must have a Promise polyfill. If you do not want to provide one, pass a callback instead.');
       }
       return new window.Promise(function(resolve, reject) {
+        if (!url) {
+          return reject(new Error('url is not defined'));
```

Comment:
👍 Much more comfortable flagging this for easier debugging rather than just tolerating it